benchmark/FindCXXStandard.cmake

```diff
@@ -16,34 +16,42 @@ file(WRITE "${SOURCE_FILE}"
     "int main() { }                                                 \n"
 )
 
-# Check for C++14
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+message(STATUS "Check for C++14")
```

Comment:
If we want to print a message, let it be something like `"C++ standard found to be ..."`, and print it only once to avoid cluttering the CMake generation step. We just have to put the message inside the `if(_supports_xx)` statement.